src/views/SystemList/index.tsx

```diff
@@ -25,29 +25,42 @@ import PageWrapper from 'components/PageWrapper';
 import Alert from 'components/shared/Alert';
 import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
 import SectionWrapper from 'components/shared/SectionWrapper';
-import GetCedarSystemsQuery from 'queries/GetCedarSystemsQuery';
+import GetCedarSystemsAndBookmarksQuery from 'queries/GetCedarSystemsAndBookmarksQuery';
 import {
-  GetCedarSystems,
-  GetCedarSystems_cedarSystems as CedarSystem
-} from 'queries/types/GetCedarSystems';
-import { mockBookmarkInfo } from 'views/Sandbox/mockSystemData'; // TODO - replace mockSystemInfo/mockBookmarkInfo with dynamic data fetched from backend and CEDAR
+  GetCedarSystemsAndBookmarks,
+  GetCedarSystemsAndBookmarks_cedarSystemBookmarks as CedarSystemBookmark,
+  GetCedarSystemsAndBookmarks_cedarSystems as CedarSystem
+} from 'queries/types/GetCedarSystemsAndBookmarks';
 
 import Table from './Table';
-import { filterBookmarks } from './util';
+import filterBookmarks from './util';
 
 import './index.scss';
 
 export const SystemList = () => {
   const { t } = useTranslation('systemProfile');
 
   // TODO: query parameters and caching
-  const { loading, error, data } = useQuery<GetCedarSystems>(
-    GetCedarSystemsQuery
-  );
+  const {
+    loading,
+    error,
+    data,
+    refetch
+  } = useQuery<GetCedarSystemsAndBookmarks>(GetCedarSystemsAndBookmarksQuery);
+
+  const systemsTableData = (data?.cedarSystems
+    ? data.cedarSystems
+    : []) as CedarSystem[];
+
+  const bookmarks = (data?.cedarSystemBookmarks
+    ? data.cedarSystemBookmarks
+    : []) as CedarSystemBookmark[];
 
-  const systemsTableData = data?.cedarSystems
-    ? (data.cedarSystems as CedarSystem[])
-    : ([] as CedarSystem[]);
+  //
+  const refetchBookmarks = () =>
+    refetch({
+      include: ['cedarSystemBookmarks']
```

Comment:
This seems to be refetching the System list as well, not just the bookmarks. I think this line is implying that it should only be doing bookmarks (and I agree, because the System fetch takes a hot minute!)

Reply:
Yeah, doesn't look like the `include` property can change the query that's being run; from the apollo-client docs, it looks the argument to `refetch()` is for passing in updated query arguments. 

I think what we could do is separate the queries for systems and bookmarks. Call them from separate `useQuery()` calls, then you can refetch the bookmarks only when they get updated.

Reply:
I separated these out into separate queries and only the bookmarks are fetched again when needed.